mods/ts/weapons/healweapons.yaml

```diff
@@ -1,25 +1,25 @@
-Heal:
+^HealWeapon:
 	ReloadDelay: 80
 	Range: 2c849
 	Report: healer1.aud
-	ValidTargets: Infantry
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
```

Comment:
All our projectiles currently pass only Target.FromPos, so the target passed through the impact is  never an actor (that's what my in-game tests suggest, anyway).

So unless I'm missing something, this would require some code gymnastics that would have to be applied to all projectiles, in addition to the new warhead itself.
Unless we just make the warhead simply perform a small-area scan for actor(s) at the impact pos.

AFAIK the original games simply used Spread=0 which is equal to "only deal 'damage' at impact pos with no spread beyond that".